src/main/java/com/faforever/moderatorclient/api/event/FafUserFailModifyEvent.java

```diff
@@ -0,0 +1,16 @@
+package com.faforever.moderatorclient.api.event;
+
+import lombok.Value;
+
+@Value
+public class FafUserFailModifyEvent {
+    Throwable cause;
+    String url;
+    Class<?> objectClass;
+
+    public FafUserFailModifyEvent(Throwable cause, Class<?> objectClass, String url) {
+        this.cause = cause;
+        this.objectClass = objectClass;
+        this.url = url;
```

Comment:
Shouldn't lombok create this one?
Maybe you should refactor the order of the arguments first, so it doesn't break anything.

Reply:
ah yeah you are correct missed this one